### PR TITLE
update favicons

### PR DESCRIPTION
**Pull request by Arhell**
fix issue https://github.com/reactiveui/website/issues/508

### CODE REVIEW

File: input/_Head.cshtml [hunk 1]
```diff
@@ -1,19 +1,12 @@
...
-        <meta name="theme-color" content="#ffffff" />
+        <link rel="apple-touch-icon" sizes="180x180" href="@Context.GetLink("/assets/img/favicons/apple-touch-icon.png")">
+        <link rel="icon" type="image/png" sizes="32x32" href="@Context.GetLink("/assets/img/favicons/favicon-32x32.png")">
+        <link rel="icon" type="image/png" sizes="16x16" href="@Context.GetLink("/assets/img/favicons/favicon-16x16.png")">
+        <link rel="manifest" href="@Context.GetLink("/assets/img/favicons/manifest.json")"/>
+        <link rel="shortcut icon" type="image/x-icon" href="@Context.GetLink("/assets/img/favicons/favicon.ico")"/>
+        <link rel="mask-icon" href="@Context.GetLink("/assets/img/favicons/safari-pinned-tab.svg")" color="#ffffff">
+        <meta name="msapplication-config" content="@Context.GetLink("/assets/img/favicons/browserconfig.xml")" />
+        <meta name="msapplication-TileColor" content="#ffffff">
+        <meta name="theme-color" content="#ffffff">
```
Comment (worldbeater) [inline input/_Head.cshtml:9]: Why remove `msapplication-TileImage`  and different dimensions of `apple-touch-icon`?

Comment (Arhell) [reply input/_Head.cshtml:9]: this is the best practice that has been generated by this service https://realfavicongenerator.net/
Also, such a number of icons is unnecessary. For IE we use browserconfig.xml

**Review by worldbeater — APPROVED** [review]
Looks good, thanks!

**Comment by glennawatson** [comment]
Thanks for helping out.

**Pull request merged**